src/components/CurrencyInput/CurrencyInput.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { withTheme } from 'emotion-theming';
+
+import IconInput from '../IconInput';
+import { themePropType } from '../../util/shared-prop-types';
+
+const currencyBaseStyles = ({ theme }) => css`
+  color: ${theme.colors.b700};
+  line-height: 16px;
+`;
+
+const CurrencyIcon = styled('span')`
+  ${currencyBaseStyles};
+`;
+
+/**
+ * SearchInput component for forms.
+ */
+const CurrencyInput = ({ currencyPosition, currency, theme, ...props }) => (
```

Comment:
There's no need to explicitly pass the theme to the `<CurrencyInput>`. It receives it automatically.